cartography/models/aws/iam/access_key.py

```diff
@@ -0,0 +1,103 @@
+from cartography.models.core.relationships import TargetNodeMatcher
+
+
+@dataclass(frozen=True)
+class AccountAccessKeyNodeProperties(CartographyNodeProperties):
+    # Required unique identifier
+    id: PropertyRef = PropertyRef("accesskeyid")
+    accesskeyid: PropertyRef = PropertyRef("accesskeyid", extra_index=True)
+
+    # Automatic fields (set by cartography)
+    lastupdated: PropertyRef = PropertyRef("lastupdated", set_in_kwargs=True)
+
+    # Business fields from AWS IAM access keys
+    createdate: PropertyRef = PropertyRef("createdate")
+    status: PropertyRef = PropertyRef("status")
+    lastuseddate: PropertyRef = PropertyRef("lastuseddate")
+    lastusedservice: PropertyRef = PropertyRef("lastusedservice")
+    lastusedregion: PropertyRef = PropertyRef("lastusedregion")
+
+
+@dataclass(frozen=True)
+class AWSUserToAccountAccessKeyRelProperties(CartographyRelProperties):
+    lastupdated: PropertyRef = PropertyRef("lastupdated", set_in_kwargs=True)
+
+
+@dataclass(frozen=True)
+class AWSUserToAccountAccessKeyRel(CartographyRelSchema):
+    target_node_label: str = "AccountAccessKey"
+    target_node_matcher: TargetNodeMatcher = make_target_node_matcher(
+        {
+            "accesskeyid": PropertyRef("accesskeyid"),
+        }
+    )
+    direction: LinkDirection = LinkDirection.OUTWARD
+    rel_label: str = "AWS_ACCESS_KEY"
+    properties: AWSUserToAccountAccessKeyRelProperties = (
+        AWSUserToAccountAccessKeyRelProperties()
+    )
+
+
+@dataclass(frozen=True)
+class AccountAccessKeyToAWSUserRelProperties(CartographyRelProperties):
+    lastupdated: PropertyRef = PropertyRef("lastupdated", set_in_kwargs=True)
+
+
+@dataclass(frozen=True)
+class AccountAccessKeyToAWSUserRel(CartographyRelSchema):
+    target_node_label: str = "AWSUser"
+    target_node_matcher: TargetNodeMatcher = make_target_node_matcher(
+        {
+            "arn": PropertyRef("user_arn"),
+        }
+    )
+    direction: LinkDirection = LinkDirection.INWARD
+    rel_label: str = "AWS_ACCESS_KEY"
+    properties: AccountAccessKeyToAWSUserRelProperties = (
+        AccountAccessKeyToAWSUserRelProperties()
+    )
+
+
+@dataclass(frozen=True)
+class AccountAccessKeyToAWSAccountRelProperties(CartographyRelProperties):
+    lastupdated: PropertyRef = PropertyRef("lastupdated", set_in_kwargs=True)
+
+
+@dataclass(frozen=True)
+class AccountAccessKeyToAWSAccountRel(CartographyRelSchema):
+    target_node_label: str = "AWSAccount"
+    target_node_matcher: TargetNodeMatcher = make_target_node_matcher(
+        {
+            "id": PropertyRef("AWS_ID", set_in_kwargs=True),
+        }
+    )
+    direction: LinkDirection = LinkDirection.INWARD
+    rel_label: str = "RESOURCE"
+    properties: AccountAccessKeyToAWSAccountRelProperties = (
+        AccountAccessKeyToAWSAccountRelProperties()
+    )
+
+
+@dataclass(frozen=True)
+class AccountAccessKeySchema(CartographyNodeSchema):
+    label: str = "AccountAccessKey"
```

Comment:
I know this is in draft so I wont leave a full review yet, but I do think the term "account access key" may be a bit misleading. My understanding is that these are all just "access keys." However, if the access key is owned by root, then it effectively gives access to the entire account, which I guess could be considered an "account access key." I think most of these objects should just be access keys: https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_access-keys.html

This might feel nit picky but I think vocab is important here given the slew of auth mechanism and key types in AWS